### PR TITLE
[webkitcorepy] Indicate to user the default option

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 93f5838f8ad8eb9934654b256f6d39d204213309
<pre>
[webkitcorepy] Indicate to user the default option
<a href="https://bugs.webkit.org/show_bug.cgi?id=233342">https://bugs.webkit.org/show_bug.cgi?id=233342</a>
&lt;rdar://problem/85573383 &gt;

Reviewed by Ryan Haddad.

* Tools/Scripts/libraries/webkitcorepy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/terminal.py:
(Terminal.choose): Surround the default option with square brackets.
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/tests/terminal_unittest.py:
(TerminalTests.test_choose_triple):
* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py:
(Setup.github): We should construct a user owned mirror by default.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/land_unittest.py:
(TestLand.test_default):
(TestLand.test_canonicalize):
(TestLand.test_svn):
(TestLandGitHub):
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/setup_unittest.py:
(TestSetup.test_github):

Canonical link: <a href="https://commits.webkit.org/244463@main">https://commits.webkit.org/244463@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@286076">https://svn.webkit.org/repository/webkit/trunk@286076</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

File: Tools/Scripts/libraries/webkitcorepy/setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def readme():
 
 setup(
     name='webkitcorepy',
-    version='0.12.1',
+    version='0.12.2',
```
Comment (akath20) [inline Tools/Scripts/libraries/webkitcorepy/setup.py]: Is there a way to sync this version with the version in `__init__.py`?

File: Tools/Scripts/libraries/webkitcorepy/webkitcorepy/terminal.py [hunk 2]
```diff
@@ -45,10 +45,15 @@ def choose(cls, prompt, options=None, default=None, strict=False, numbered=False
         response = None
         while response is None:
             if numbered:
-                numbered_options = ['{}) {}'.format(i + 1, options[i]) for i in range(len(options))]
+                numbered_options = ['{}) {}'.format(
```
Comment (akath20) [inline Tools/Scripts/libraries/webkitcorepy/webkitcorepy/terminal.py]: Can this be on the same line as the one below? Or making these into variables on their own line to make them easier to read?

**Review by akath20 — APPROVED** [review]

File: Tools/Scripts/libraries/webkitcorepy/setup.py (hunk 1 above)
Comment (JonWBedard) [reply Tools/Scripts/libraries/webkitcorepy/setup.py]: Yes, but it's not trivial. I have a partial change to do it, but it's not particularly high-priority at the moment.

**Review by ryanhaddad — APPROVED** [review]

**Comment by JonWBedard** [comment]
Landed https://commits.webkit.org/244463@main (93f5838f8ad8eb9934654b256f6d39d204213309)!

**Pull request merged**